### ISSUE TITLE
plugin not loading even  from mer testing releases

### ISSUE THREAD

**Issue by PROSKAAAAA** [Michal78900/MapEditorReborn#98]
i use dedicated server with exiled                        
![Снимок экрана 2024-07-11 100643](https://github.com/Michal78900/MapEditorReborn/assets/167566308/2ad19251-36c0-4b6c-b441-30cd7385e629)

**Comment by PROSKAAAAA** [comment]
any help?

**Comment by Maciupek** [comment]
What Exiled & Mer version are you using?

**Comment by PROSKAAAAA** [comment]
Im away from pc rn i wil be back in 4 hours

**Comment by PROSKAAAAA** [comment]
one sec

**Comment by PROSKAAAAA** [comment]
![Снимок экрана 2024-07-21 192224](https://github.com/user-attachments/assets/456a0b68-5be9-4b32-893c-42bbba74df2e)
![Снимок экрана 2024-07-21 192331](https://github.com/user-attachments/assets/786dc68d-7c35-4858-8c4c-e3ec0292a9fe)
version of mer is the latest from testing releases and its not giving a error and i even cant join my server because of versions

**Comment by PROSKAAAAA** [comment]
hey

**Comment by Maciupek** [comment]
You shouldn't use Resonance for now, especially for a public server. mer doesn't support Exiled Resonance

**Comment by PROSKAAAAA** [comment]
so how do i disable it?

**Comment by Maciupek** [comment]
Change youre Exiled Version? And update youre game to 13.5.1

**Comment by PROSKAAAAA** [comment]
game is updated but how to сhange exiled version?

**Comment by Maciupek** [comment]
Just download it from the github

**Comment by PROSKAAAAA** [comment]
![Снимок экрана 2024-07-22 181017](https://github.com/user-attachments/assets/189e934e-0842-4639-8163-5ef45ba75be6)
this version?

**Comment by PROSKAAAAA** [comment]
and there are alpha releases too

**Comment by Maciupek** [comment]
Yes, dont use Exiled 9 until it fully comes out I would recommend

**Comment by PROSKAAAAA** [comment]
![Снимок экрана 2024-07-22 190122](https://github.com/user-attachments/assets/507f7078-cfa3-4ae6-9cdf-2c63acff6475)
exiled even didnt generated configs for mer also i cant join my server because it says  the game  version is diffrent from the server

**Comment by PROSKAAAAA** [comment]
ahh im so tired

**Comment by PROSKAAAAA** [comment]
what should i do?

**Comment by Maciupek** [comment]
Read the error message, you need to update youre game to 13.5.1

**Comment by PROSKAAAAA** [comment]
its already updated

**Comment by Maciupek** [comment]
Type version/ver in youre console if its 13.5.0 then its not

**Comment by PROSKAAAAA** [comment]
Im away from pc rn

**Comment by PROSKAAAAA** [comment]
Im will be back in 5 days

**Comment by PROSKAAAAA** [comment]
![Снимок экрана 2024-07-31 213000](https://github.com/user-attachments/assets/64f2a285-4b44-42f8-9e49-10ea3aa5ebbd)

**Comment by Maciupek** [comment]
As it states in the console message youre game is not updated, and is still on 13.5.0

**Comment by PROSKAAAAA** [comment]
okey how do i update my game

**Comment by PROSKAAAAA** [comment]
scp sl says my game is already 13.5.1

**Comment by Maciupek** [comment]
The server is not updated, not youre game

**Comment by PROSKAAAAA** [comment]
so how do i update the server?

**Comment by PROSKAAAAA** [comment]
hey?

**Comment by PROSKAAAAA** [comment]
heeyyy?

**Comment by PROSKAAAAA** [comment]
hello?

**Comment by PROSKAAAAA** [comment]
heyyy?

**Comment by PROSKAAAAA** [comment]
im gonna be away from pc for 20 days

**Comment by PROSKAAAAA** [comment]
im away from pc

**Comment by PROSKAAAAA** [comment]
Heyy help

**Comment by PROSKAAAAA** [comment]
> The server is not updated, not youre gams

sooo how do i update my server&
